external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/package.scala

```diff
@@ -33,8 +33,8 @@ package object kafka010 {   // scalastyle:ignore
       .createWithDefaultString("10m")
 
   private[kafka010] val CONSUMER_CACHE_CAPACITY =
```

Comment:
Thank you for new doc, @gaborgsomogyi .
Since this is used since Spark 2.0.x, shall we add a comment on the migration doc?

Reply:
Originally I've taken a look at the related migration guide (sql-migration-guide-upgrade) but there I haven't seen any parameter rename entry. Thought it's the way because of the following:
* The parameter just deprecated and will work properly in 3.0 as well
* The Spark config system will give a warning message which tells what is the new parameter

If my thoughts are not correct please correct it and I'll comment on the migration doc.

Reply:
Yeah this one doesn't require migration per se. We'd definitely mention it if the old config went away. Could mention it now too. My only hesitation is overloading the already huge 3.0 migration guide with something that doesn't actually require action now and will be flagged to the user.

Reply:
Same understanding.

Reply:
Got it. Thanks!